elastic/kibana
# [Entity Analytics][Entity Store] Init: Put engine in error state if data view does not exist

## Summary

Fixes a bug where if the securite default data view does not exist the entity engine would get stuck in 'installing' status. After this fix, the engine is put into 'error' state.

The entity engine uses the security default data view by default to generate it's source index pattern.

## Testing

1. Delete the security default data view `security-solution-default`
2. Init the entity store
3. Observe that the entity store is in an error state (the UI should show the error)
